.github/workflows/ci.yml

```diff
@@ -0,0 +1,15 @@
+name: ci 
+on:
+  push:
+    branches:
+      - main
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - run: pip install mkdocs-material 
```

Comment:
at this point, am thinking we may need to separate the `requirements.txt` for documentation from that of the rest of the project, so that we install all the packages required for docs from `requirements.txt` by `run: pip install -r requirements.txt`

Reply:
@HassanBahati just a minor issue, but the actions say you might be using a deprecated component in the workflow. Perhaps it's something you wanna look at.